src/components/docs/Cards/ColorCard/index.js

```diff
@@ -9,13 +9,13 @@ import classnames from 'classnames';
 import styles from './styles.module.css';
 
 const ColorCard = ({ color, icon, overlay, title, to, type, ...props }) => (
-  <BaseContainer 
+  <BaseContainer
     className={classnames(styles.root, styles[color], {
       [styles.snippetTab]: type === 'snippetTab',
       [styles.withOverlay]: type === 'snippetTab' && overlay,
-    })} 
+    })}
     hasShadow={false}
-    tabIndex={type === 'snippetTab' && -1}
+    tabIndex={`"${type === 'snippetTab' && -1}"`}
```

Comment:
`tabIndex` accepts a string.

Reply:
`type === 'snippetTab' && "-1"` seems simplest to me